app/src/main/java/tttTDD/Java/Game.java

```diff
@@ -0,0 +1,48 @@
+package tttTDD.Java;
+
+import tttTDD.Java.Interfaces.Player;
+
+public class Game {
+    Board board;
+    Player player1;
+    Player player2;
+    Player currentPlayer;
+    boolean gameCompleted;
+    String gameWinner;
```

Comment:
All of these fields are implicitly public to any class in the package. Is this intentional?

For example, is it desirable for methods outside of the `Game` to be able to change the value of `gameCompleted`?

Reply:
I like the call out here around Game, the games state, and a GameState class. -- I would give pause to a need that extracts private instance variables into another class without specific behavior on the new class that conflicts with the origin place of the variables.

Reply:
I agree, I dont think we need to introduce a `GameState` object at this point. I think these should be explicitly private and if this information is needed, it should be encapsulated in a method

Reply:
Ah, cool, didn't know Java did that. -- Let's specifically mark those as private then.